packages/xo-server-cloud/src/index.js

```diff
@@ -118,14 +120,16 @@ class XoServerCloud {
 
   // ----------------------------------------------------------------
 
-  async _requestResource(namespace, id, version) {
+  async _requestResource(namespace, id, version, hub) {
```

Comment:
Avoid too many params and avoid (implicit) boolean params, you should convert this method to use a param object for named params, example:

```suggestion
  async _requestResource({ namespace, id, version, hub = false }) {
```